doc/man1/mac.pod

```diff
@@ -0,0 +1,152 @@
+=pod
+
+=head1 NAME
+
+openssl-mac,
+mac - perform Message Authentication Code operations
+
+=head1 SYNOPSIS
+
+B<openssl mac>
+B<-name <mac_name>>
+[B<-help>]
+[B<-ctrl>]
+[B<-bin>]
+[B<-out filename>]
+[B<file>]
+
+B<openssl> I<mac> B<-name <mac_name>>  [B<...>]
+
+=head1 DESCRIPTION
+
+The message authentication code functions output the MAC of a supplied file
+in hexadecimal.
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<-help>
+
+Print out a usage message.
+
+=item B<-name <mac_name>>
+
+Specifies name of a supported mac algorithm which will be used. To see the list
+of supported MAC's use the command I<list -mac-algorithms>. This option must
+be specified.
+
+=item B<-bin>
+
+Output the MAC in binary form. Uses hexadecimal text format if not specified. 
+
+=item B<-out filename>
+
+Filename to output to, or standard output by default.
+
+=item B<-ctrl nm:v>
+
+Passes options to the MAC algorithm.
+The following options are supported:
+
+=over 4
+
+=item B<key:string>
+
+Specifies the MAC key as an alphanumeric string (use if the key contains
+printable characters only). The string length must conform to any restrictions
+of the MAC algorithm. A key must be specified for any MAC algorithm.
```

Comment:
What do you mean by "A key must be specified for _any_ MAC algorithm."? Do you mean "A key must be specified for _every_ MAC algorithm."?

Reply:
yes, its an option that always needs to be specified. Changed to every.